### DISCUSSION TITLE
Why is the `Eclipse Theia v1.40.1 Latest` release different from other releases?

### DISCUSSION THREAD

**Discussion by kittaakos** [eclipse-theia/theia#12880] [Q&A] [answered]
Hi There,

Why is the latest release (`Eclipse Theia v1.40.1 Latest`: https://github.com/eclipse-theia/theia/releases/tag/v1.40.1) different than others? I go to the release page; one commit is mentioned compared to other releases with changelog, breaking changes, etc.. I click on the link (https://github.com/eclipse-theia/theia/releases/tag/v1.34.1) in the release description and get a 404.

Thanks!

----

_Update:_

This is what I do not understand:

![Screenshot 2023-08-31 at 13 09 55](https://github.com/eclipse-theia/theia/assets/1405703/fb01f479-30b8-40c4-acfd-7338d201585d)

If I click on the changes, I get this changeset: https://github.com/eclipse-theia/theia/compare/v1.40.1...master
If I compare the release with the previous one, I get this changeset: https://github.com/eclipse-theia/theia/compare/v1.40.0...v1.40.1

**Comment by msujew** [comment, marked as answer]
@tsmaeder Looks like the release description is pretty inconsistent. Copy & Paste error?

@kittaakos Note that 1.40.1 is just a community release fixing some bugs that would only be available in 1.41.0 otherwise.

**Reply by kittaakos** [reply]
I updated my question. I would like to know what's in the release.

Thanks for your great work 🙌

**Reply by kittaakos** [reply]
> available in 1.41.0 otherwise.

Mark, do you have a scheduled release date?

**Reply by msujew** [reply]
Yes, today. Last Thursday of the month, as usual :)

**Reply by tsmaeder** [reply]
Since the last release was 1.40.0, I figured this is now our "latest" release. Unfortunately, the release instructions do not says anything on the topic. The problem should fix itself later today anyway, when we release 1.41.0, right?
